src/framapp-polymer-app/login-component.html

```diff
@@ -0,0 +1,62 @@
+<link rel="import" href="../../bower_components/polymerfire/firebase-app.html">
+<link rel="import" href="../../bower_components/polymerfire/firebase-auth.html">
+<link rel="import" href="../../bower_components/paper-dialog/paper-dialog.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../../bower_components/iron-icons/communication-icons.html">
+
+<dom-module id="login-component">
+    <template>
+      <style>
+
+      </style>
+      <firebase-app
+        auth-domain="project-5613440220430148247.firebaseapp.com"
+        database-url="https://project-5613440220430148247.firebaseio.com"
+        api-key="AIzaSyBKP4cOP508h0JLKmjFvzJooO0MqV8l4fU">
+      </firebase-app>
+      <firebase-auth id="auth" user="{{user}}" provider="google" on-error="handleError">
+      </firebase-auth>
+      <paper-button onclick="loginDialog.open()">Login</paper-button>
+      <paper-dialog id="loginDialog">
+        <h2>Insert login data</h2>
+        <div>
+          <paper-input label="Username" id="userName">
```

Comment:
Ok, if you say that this approach is correct, then nothing is wrong. But I just don't like ```ID's``` ;)